client/js/views/Stats.vue

```diff
@@ -0,0 +1,151 @@
+            {{ item.name }}
+          </h1>
+        </template>
+      </v-select>
+      <h4 v-if="selectedSeason" class="text-h4">
+        <span data-cy="season-start-date">{{ seasonStartString }}</span>
+        -
+        <span data-cy="season-end-date"> {{ seasonEndString }} </span>
+      </h4>
+    </section>
+    <section class="px-8">
+      <!-- Season Champions -->
+      <div class="mb-10">
+        <h2
+          v-if="
+            selectedSeason &&
+              (selectedSeason.bracketLink ||
+                selectedSeason.firstPlace ||
+                selectedSeason.secondPlace ||
+                selectedSeason.thirdPlace)
+          "
+          class="text-h2 mb-4"
+        >
+          Season Champions
+        </h2>
+        <div class="d-flex justify-space-around flex-wrap">
+          <award-card
+            v-if="selectedSeason && selectedSeason.firstPlace"
+            :username="selectedSeason.firstPlace"
+            :place="1"
+            class="mb-4"
+          />
+          <award-card
+            v-if="selectedSeason && selectedSeason.secondPlace"
+            :username="selectedSeason.secondPlace"
+            :place="2"
+            class="mb-4"
+          />
+          <award-card
+            v-if="selectedSeason && selectedSeason.thirdPlace"
+            :username="selectedSeason.thirdPlace"
+            :place="3"
+            class="mb-4"
+          />
+        </div>
+        <p v-if="selectedSeason && selectedSeason.bracketLink" class="text-body-1">
+          Click
+          <a :href="selectedSeason.bracketLink" data-cy="tournament-bracket-link">
+            here to see the official tournament bracket
+          </a>
+        </p>
+        <p v-if="selectedSeason && selectedSeason.footageLink" class="text-body-1">
+          Click
+          <a :href="selectedSeason.footageLink" data-cy="tournament-footage-link">
+            here to watch the official tournament footage
+          </a>
+          with play-by-play commentary.
+        </p>
+      </div>
+      <!-- Rankings Table -->
+      <h2 class="text-h2 ml-4">
+        Weekly Rankings
+        <stats-scoring-dialog />
+      </h2>
+      <stats-leaderboard :loading="loadingData" :season="selectedSeason" />
+    </section>
+  </div>
+</template>
+
+<script>
+import { io } from '@/plugins/sails.js';
+import AwardCard from '@/components/AwardCard.vue';
+import StatsLeaderboard from '@/components/StatsLeaderboard.vue';
+import StatsScoringDialog from '@/components/StatsScoringDialog.vue';
+
+const dayjs = require('dayjs');
+
+export default {
+  name: 'Stats',
+  components: {
+    AwardCard,
+    StatsLeaderboard,
+    StatsScoringDialog,
+  },
+  data() {
+    return {
+      loadingData: false,
+      selectedSeason: null,
+      seasons: [],
+    };
+  },
+  computed: {
+    seasonStartString() {
+      if (this.selectedSeason) {
+        return dayjs(this.selectedSeason.startTime).format('YYYY/MM/DD');
+      }
+      return '';
```

Comment:
I would probably flip these to handle the error case first:
```suggestion
      if (!this.selectedSeason) {
        return '';
      }
      return dayjs(this.selectedSeason.startTime).format('YYYY/MM/DD');
```

Alternatively, just combine it in to one return:
```suggestion
      return !this.selectedSeason
        ? ''
        : dayjs(this.selectedSeason.startTime).format('YYYY/MM/DD');
```
Trying to figure out how prettier would format something is 🤮 lol.

Reply:
Yeah I generally prefer the early return via ternary or otherwise. Something that concerns me about it though is that when we move to typescript, I'm medium confident that it's not smart enough to detect early returns and will throw a fit. 

Probably an issue worth addressing when we get there. For now I'll take the ternary